integration/query_array_test.go

```diff
@@ -406,6 +247,8 @@ func TestQueryArrayAll(t *testing.T) {
 	// Insert additional data to check more complicated cases:
 	// - a longer array of ints;
 	// - a field is called differently and needs to be found with the {$all: [null]} case.
+	// TODO Add "many-integers" to shareddata.Composites once more
```

Comment:
Do we need an issue number for this `TODO`, or is it in the scope of the current task?

Reply:
It's in scope of the current task, it will be in upcoming PR :hugs: